resources/function_help/json/overlay_intersects

```diff
@@ -28,6 +28,16 @@
       "description": "set this to true to build a local spatial index (most of the time, this is unwanted, unless you are working with a particularly slow data provider)",
       "optional": true,
       "default": "false"
+    },
+    {
+      "arg": "min_overlap",
+      "description": "for polygons an optional minimum area in current feature squared units for the intersection (if the intersection results in multiple polygons the intersection will be returned if at least one of the polygons has an area greater or equal to the value), for lines an optional minimum length in current feature units (if the intersection results in multiple lines the intersection will be returned if at least one of the lines has a length greater or equal to the value).",
+      "optional": true
+    },
+    {
+      "arg": "min_inscribed_circle_radius",
+      "description": "for polygons only an optional minimum radius in current feature units for the maximum inscribed circle of the intersection (if the intersection results in multiple polygons the intersection will be returned if at least one of the polygons has a radius for the maximum inscribed circle greater or equal to the value).<br>Read more on the underlying GEOS predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_MaximumInscribedCircle.html'>ST_MaximumInscribedCircle</a> function.",
```

Comment:
There's still a GEOS mention here

Reply:
> 

as there is in all other functions:

```
$ grep -r GEOS resources/function_help/
resources/function_help/json/overlay_crosses:  "description": "Returns whether the current feature spatially crosses at least one feature from a target layer, or an array of expression-based results for the features in the target layer crossed by the current feature.<br><br>Read more on the underlying GEOS \"Crosses\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Crosses.html'>ST_Crosses</a> function.",
resources/function_help/json/overlay_disjoint:  "description": "Returns whether the current feature is spatially disjoint from all the features of a target layer, or an array of expression-based results for the features in the target layer that are disjoint from the current feature.<br><br>Read more on the underlying GEOS \"Disjoint\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Disjoint.html'>ST_Disjoint</a> function.",
resources/function_help/json/overlay_intersects:  "description": "Returns whether the current feature spatially intersects at least one feature from a target layer, or an array of expression-based results for the features in the target layer intersected by the current feature.<br><br>Read more on the underlying GEOS \"Intersects\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Intersects.html'>ST_Intersects</a> function.",
resources/function_help/json/overlay_intersects:      "description": "for polygons only an optional minimum radius in current feature units for the maximum inscribed circle of the intersection (if the intersection results in multiple polygons the intersection will be returned if at least one of the polygons has a radius for the maximum inscribed circle greater or equal to the value).<br>Read more on the underlying GEOS predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_MaximumInscribedCircle.html'>ST_MaximumInscribedCircle</a> function.",
resources/function_help/json/overlay_within:  "description": "Returns whether the current feature is spatially within at least one feature from a target layer, or an array of expression-based results for the features in the target layer that contain the current feature.<br><br>Read more on the underlying GEOS \"Within\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Within.html'>ST_Within</a> function.",
resources/function_help/json/overlay_contains:  "description": "Returns whether the current feature spatially contains at least one feature from a target layer, or an array of expression-based results for the features in the target layer contained in the current feature.<br><br>Read more on the underlying GEOS \"Contains\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Contains.html'>ST_Contains</a> function.",
resources/function_help/json/overlay_equals:  "description": "Returns whether the current feature spatially equals to at least one feature from a target layer, or an array of expression-based results for the features in the target layer that are spatially equal to the current feature.<br><br>Read more on the underlying GEOS \"Equals\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Equals.html'>ST_Equals</a> function.",
resources/function_help/json/overlay_touches:  "description": "Returns whether the current feature spatially touches at least one feature from a target layer, or an array of expression-based results for the features in the target layer touched by the current feature.<br><br>Read more on the underlying GEOS \"Touches\" predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_Touches.html'>ST_Touches</a> function.",
```

Reply:
If I may chime in here: I think adding a reference to PostGIS or GEOS documentation (if that resource seems stable) and the documentation is "understandable" by the average user does seem like a good idea. GEOS might be an edge case, as this is really very technical. But the PostGIS docs are sometimes really useful also for less experienced users.

@elpaso and I also discussed that it would be really useful if a future version of the QGIS expression docs would allow for images within the documentation. Often an image would illustrate things better than just text. Maybe something for a 2022 grant ...

Reply:
Regarding images, it's probably already possible because the help string is HTML, it would take some time to prepare the images, adjust CMake to compile the resources and make sure they are formatted nicely though. Providing a link to postgis is certainly a simple but effective approach in the meantime.

Reply:
> I think adding a reference to PostGIS or GEOS documentation 
> GEOS might be an edge case, as this is really very technical. But the PostGIS docs are sometimes really useful also 

I agree entirely -- I think linking to PostGIS' documentation is fine, but GEOS docs aren't useful for non-developers and I can't see any point to linking there.

To me GEOS is an implementation detail, and it's not relevant to mention it at all for users. (Just like the data source manager has tabs for "vector" and "raster" instead of "ogr" and "gdal"!). It's just a technical term used when it doesn't need to be which has the downside that it increases the barrier of entry and makes things seem more complex then they are.

Reply:
I'm not going to change all other GEOS mentions in this PR, for consistency it's better to leave it as it is. We can purge GEOS from all other help text in a separate PR, provided that there is consent on that.

Reply:
Marking resolved because there was never a link to GEOS, the link was (and is) to postgis.